helm/fdb-operator/templates/rbac/rbac_role_binding.yaml

```diff
@@ -1,13 +1,24 @@
 apiVersion: rbac.authorization.k8s.io/v1
+{{- if .Values.operator.globalMode.enabled }}
+kind: ClusterRoleBinding
+{{- else }}
 kind: RoleBinding
+{{- end }}
 metadata:
   name: {{ .Values.operator.rolebinding }}
   labels:
     {{- include "chart.labels" . | nindent 4 }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
+  {{- if .Values.operator.globalMode.enabled }}
+  kind: ClusterRole
+  {{- else }}
   kind: Role
+  {{- end }}
   name: {{ .Values.operator.role }}
 subjects:
 - kind: ServiceAccount
   name: fdb-kubernetes-operator-controller-manager
+  {{- if .Values.operator.globalMode.enabled }}
```

Comment:
I think we can put this here as a default and remove the if statement

Reply:
Done!